docker/Dockerfile

```diff
@@ -21,8 +21,15 @@ WORKDIR /app
 COPY . /app
 
 RUN bundle
+
+# Precompile assets
+RUN RAILS_ENV=production SECRET_KEY_BASE=precompile_placeholder bundle exec rake assets:precompile
```

Comment:
Maybe `RAILS_ENV` should be set with the docker `ENV` statement, otherwise it may run with `development` env if not set at runtime?

Reply:
This env is only for `rake asset:precompile`. This would not set RAILS_ENV globally

Reply:
@kolaente We are setting RAILS_ENV globally to build the docker images from docker hub. You can override this env variable via docker-compose. Please check out docker-compose.yml if you like to use docker for development.